test/src/e2e_vm_tests/mod.rs

```diff
@@ -218,12 +219,24 @@ fn discover_test_configs() -> Result<Vec<TestDescription>, String> {
     Ok(configs)
 }
 
+const DIRECTIVE_RX: &str = r"\b#\s*(\w+):\s+(.*)";
```

Comment:
This _could_ be stricter by insisting the comment is the only thing on the line, with `^\s*#...` but I guess in a TOML file it'd be pretty hard to actually be a mismatch.